website/manage.py

```diff
@@ -11,7 +11,9 @@
 
 if __name__ == "__main__":
     os.environ.setdefault("DJANGO_SETTINGS_MODULE", "website.settings")
-    sys.path.append(os.path.dirname(os.getcwd()))
```

Comment:
This here is what was causing me problems.  It changes the path based upon the CWD (`os.getcwd`).  So the directory appended to the path changes if you invoke it like `python manage.py` vs. `python website/manage.py` because the CWD is different in those two cases.